docs/07-templated-components.md

```diff
@@ -1 +1,246 @@
 # Templated components
+
+Let's refactor some of original components and make them more reusable. Along the way we'll also create a separate library project as a home for the new components.
+
+## Creating a component library (Visual Studio)
```

Comment:
TBH the instructions might be easier if we told everyone to use command line, not VS, for this, and did it with `dotnet new blazorlib`. The one necessary prerequisite is running `dotnet new --install Microsoft.AspNetCore.Blazor.Templates::0.7.0`.

Reply:
FYI I took this approach, and I left in the section where I explain what's in the project file. I already took the trouble to explain it so I decided to leave it in.